spec/marqeta/transaction_spec.rb

```diff
@@ -64,7 +64,7 @@
         expect(transactions.map(&:state)).to eq(%w[PENDING DECLINED])
         expect(transactions.map(&:user_token)).to eq(%w[user_token1 user_token2])
         expect(transactions.map(&:amount)).to eq([1000, 2000])
-        expect(transactions.map(&:created_time)).to eq(['2018-01-01T00:00:00Z', '2018-01-02T00:00:00Z'])
+        expect(transactions.map(&:created_time)).to eq([Time.parse('2018-01-01T00:00:00Z'), Time.parse('2018-01-02T00:00:00Z')])
```

Comment:
Minor, but beware of the test closely mirroring the implementation. For example, rather than `Time.parse(...`, we could instantiate a new `Time` instance with `Time.new(2018, 1, 1, 0, 0, 0, 0)`.

http://jasonrudolph.com/blog/2008/07/30/testing-anti-patterns-the-ugly-mirror/

Reply:
@andyw8 Makes sense!